lib/common/ipc_server.c

```diff
@@ -707,7 +752,7 @@ pcmk__ipc_send_xml(pcmk__client_t *c, uint32_t request, const xmlNode *message,
 
     iov_buffer = g_string_sized_new(1024);
     pcmk__xml_string(message, 0, iov_buffer, 0);
-    rc = pcmk__ipc_prepare_iov(request, iov_buffer, &iov, NULL);
+    rc = pcmk__ipc_prepare_iov(request, iov_buffer, 0, &iov, NULL);
 
     if (rc == pcmk_rc_ok) {
```

Comment:
Ditto. This needs to also handle `pcmk_rc_ipc_more`. That's basically Coverity's complaint.

Reply:
This code gets rearranged a lot in a later patch, which made this a little less obvious to see.  But, same answer here - I can add similar temporary error handling here for `pcmk_rc_ipc_more` with a comment to replace it in the next PR.